front-end/src/components/newsfeed-view/news-list/NewsList.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import  _  from 'lodash';
+import { formatDate } from 'utils/utils';
+import pic from 'assets/images/cutiepie.jpg';
+import './newslist.css';
+import NewsCardGroup from './NewsCardGroup';
+
+const dummyfeed = [
```

Comment:
Maybe lets have react-storybook or react-cosmos to have a structured way to do mocking